features/in_onderzoek.feature

```diff
@@ -27,7 +27,8 @@ Functionaliteit: in onderzoek
     Dan is in het antwoord naam.inOnderzoek.geslachtsnaam=true
     En is in het antwoord naam.inOnderzoek.voornamen=true
     En is in het antwoord naam.inOnderzoek.voorvoegsel=true
-    En is in het antwoord naam.inOnderzoek.adellijkeTitel_predikaat=true
+    En is in het antwoord naam.inOnderzoek.adellijkeTitel=true
+    En is in het antwoord naam.inOnderzoek.predikaat=true
```

Comment:
Dit kan toch niet voorkomen? Ik begrijp dat wanneer een categorie in onderzoek is, alle attributen in onderzoek zijn. Je kunt niet zowel een adellijke titel of een predicaat hebben, en dus kunnen deze ook niet beiden in onderzoek zijn. 
Is het dan zo dat er sprake is van beiden, alleen hetgeen wordt geleverd (of adellijke titel, of predicaat ) in onderzoek is?

Reply:
In de featere in_onderzoek.feature staat in de functionaliteit het volgende beschreven :

  _Wanneer een categorie in zijn geheel in onderzoek is, krijgt in het antwoord in inOnderzoek elk attribuut van deze categorie de waarde true.
  Wanneer een categorie in zijn geheel in onderzoek is, krijgt in het antwoord in  inOnderzoek in elke groep van de categorie elk attribuut van deze categorie de waarde true._ 

Dit komt dus alleen voor als indicatie dat de hele categorie in onderzoek is. Daar hebben we geen aparte inOnderzoek property voor.

Reply:
> Je kunt niet zowel een adellijke titel of een predicaat hebben, en dus kunnen deze ook niet beiden in onderzoek zijn.

> Dit komt dus alleen voor als indicatie dat de hele categorie in onderzoek is.

Ook wanneer alleen adellijkeTitel_predikaat in onderzoek staat, zal de API beide teruggeven:
    En is in het antwoord naam.inOnderzoek.adellijkeTitel=true
    En is in het antwoord naam.inOnderzoek.predikaat=true

Stel de persoon heeft een adellijke titel en die is in onderzoek, dan kan het onderzoek tot gevolg hebben dat:
- de adellijke titel wel juist is
- de adellijke titel niet juist is: geen adellijke titel en ook geen predikaat
- de adellijke titel niet juist is: geen adellijke titel maar wel een predikaat

Werkt hetzelfde als wanneer bijvoorbeeld de groep naam in onderzoek staat, terwijl de persoon geen voorvoegsel heeft. Dan wordt ook naam.inOnderzoek.voorvoegsel=true opgenomen.

Zelfde bij verblijfplaats, wanneer die in het geheel in onderzoek staat, zijn zowel postcode als locatiebeschrijving als verblijfBuitenland in onderzoek. Een persoon kan niet alle drie tegelijk hebben.